tests/e2e/main.go

```diff
@@ -293,7 +335,9 @@ func getTestCases(selectedPredefinedTests, selectedTestFiles TestSet, providerVe
 			"partial-set-security-validator-set-cap", "partial-set-security-validators-power-cap",
 			"partial-set-security-validators-allowlisted", "partial-set-security-validators-denylisted",
 			"partial-set-security-modification-proposal",
-			"active-set-changes",
+			"active-set-changes", "inactive-vals-topN",
```

Comment:
1. What about the `bestcase`s shown above?
2. Can you reorder to follow the same order as the ones above (e.g., first `"inactive-provider-validators-on-consumer"` and then `"inactive-vals-topN"`)?

Reply:
1. I think the basecases should not be run by default. They are sanity checks that tbh were mostly for me during testing, and for others to be able to confirm how it would work without the inactive vals feature
2. Sure